client/extensions/woocommerce/app/products/product-form.js

```diff
@@ -2,15 +2,12 @@
  * External dependencies
  */
 import React, { Component, PropTypes } from 'react';
-import i18n from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
-import FoldableCard from 'components/foldable-card';
 import ProductFormDetailsCard from './product-form-details-card';
-import ProductVariationTypesForm from './product-variation-types-form';
-import FormToggle from 'components/forms/form-toggle';
+import ProductFormVariationCard from './product-form-variation-card';
```

Comment:
Changed in https://github.com/Automattic/wp-calypso/pull/13155/commits/1d99a0df8199f86387e28865b5964e41993cdde3